spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/web/SleuthWebSpan.java

```diff
@@ -78,4 +78,74 @@ public String getKey() {
 
 	}
 
+	/**
+	 * Tags related to security.
+	 *
+	 * @author Marcin Grzejszczak
+	 * @since 3.1.0
+	 */
+	enum SecurityTags implements TagKey {
+
+		/**
+		 * Authorities assigned to the user.
+		 */
+		AUTHORITIES {
+			@Override
+			public String getKey() {
+				return "security.authentication.authorities";
```

Comment:
I'm thinking if we should make this configurable (enable/disable) in case anyone is concerned that breaking into the tracing backend could give extra information to attackers.